indexer/src/main/java/uk/ac/ebi/uniprot/indexer/subcell/SubcellularLocationStatisticsReader.java

```diff
@@ -0,0 +1,36 @@
+package uk.ac.ebi.uniprot.indexer.subcell;
+
+import lombok.Getter;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * @author lgonzales
+ * @since 2019-07-12
+ */
+public class SubcellularLocationStatisticsReader implements RowMapper<SubcellularLocationStatisticsReader.SubcellularLocationCount> {
+
+    @Override
+    public SubcellularLocationCount mapRow(ResultSet resultSet, int rowIndex) throws SQLException {
```

Comment:
if one of the `resultSet.get*` methods returns null, is that okay? If `SubcellularLocationCount` has null fields, will it break clients of it?

Reply:
I guess we could have a very basic test for this class to ensure that the `SubcellularLocationCount` is created correctly?

Reply:
It is a count query grouped by ID. So, I do not think it is possible to have null values, I am creating the tests